apps/gpc-prover-client/app/hooks/useIdentity.ts

```diff
@@ -1,18 +1,18 @@
 import { useEffect, useState } from "react";
-import { Identity } from "@semaphore-protocol/identity";
+import { Identity } from "semaphore-identity-v4";
 
 const useIdentity = () => {
   const [identity, setIdentity] = useState<Identity>();
 
   useEffect(() => {
-    const id = localStorage.getItem("semaphoreID");
-    if (id) {
-      setIdentity(new Identity(id));
+    const privatekey = localStorage.getItem("semaphoreID");
```

Comment:
Anyone with existing local storage is going to have to clear it for this to work.
That's probably fine (it's very few people), though it might be cleanest to rename the field to "semaphorePrivateKey" to avoid the issue entirely.

Reply:
Good point.